### PR TITLE
{KeyVault} Bump azure-mgmt-keyvault to 8.0.0

### PR DESCRIPTION
**Pull request by bim-msft**
Bump `azure-mgmt-keyvault` from `7.0.0b3` to `8.0.0`

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/keyvault/custom.py [hunk 1]
```diff
@@ -396,7 +396,7 @@ def recover_hsm(cmd, client, hsm_name, resource_group_name, location, no_wait=Fa
         resource=cmd.cli_ctx.cloud.endpoints.active_directory_graph_resource_id)
 
     parameters = ManagedHsm(location=location,
-                            sku=ManagedHsmSku(name='Standard_B1'),
+                            sku=ManagedHsmSku(name='Standard_B1', family='B'),
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/keyvault/custom.py:399]: can user specify the sku?

File: src/azure-cli/requirements.py3.Darwin.txt [hunk 2]
```diff
@@ -48,7 +48,7 @@ azure-mgmt-imagebuilder==0.4.0
 azure-mgmt-iotcentral==3.0.0
 azure-mgmt-iothub==0.12.0
 azure-mgmt-iothubprovisioningservices==0.2.0
-azure-mgmt-keyvault==7.0.0b3
+azure-mgmt-keyvault==8.0.0
```
Comment (yungezz) [inline src/azure-cli/requirements.py3.Darwin.txt:51]: is there any change required in other command modules who depends on keyvault sdk?

File: src/azure-cli/azure/cli/command_modules/keyvault/custom.py (hunk 1 above)
Comment (bim-msft) [reply src/azure-cli/azure/cli/command_modules/keyvault/custom.py:399]: @yungezz No, this is a fixed value, for now, users don't have to know this.

File: src/azure-cli/requirements.py3.Darwin.txt (hunk 2 above)
Comment (bim-msft) [reply src/azure-cli/requirements.py3.Darwin.txt:51]: @yungezz The only breaking change is SKU family, I didn't find any usages of this out of the keyvault module. Related tests on other modules have been re-recorded.

**Review by fengzhou-msft — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]
Approve for ARM part

**Review by haroldrandom — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/keyvault/tests/hybrid_2018_03_01/test_keyvault_commands.py [hunk 3]
```diff
@@ -311,6 +325,9 @@ def test_keyvault_secret(self, resource_group):
         self.kwargs['bak_file'] = bak_file
         self.cmd('keyvault secret backup --vault-name {kv} -n {sec} --file {bak_file}')
         self.cmd('keyvault secret delete --vault-name {kv} -n {sec}')
+        time.sleep(10)
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/keyvault/tests/hybrid_2018_03_01/test_keyvault_commands.py:328]: Not reliable.

Comment (bim-msft) [reply src/azure-cli/azure/cli/command_modules/keyvault/tests/hybrid_2018_03_01/test_keyvault_commands.py:328]: @qwordy Yes, just a value based on experience.

**Review by qwordy — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**